rc-events-service/src/main/scala/com/epam/random_coffee/events/repo/EventRepository.scala

```diff
@@ -0,0 +1,19 @@
+package com.epam.random_coffee.events.repo
+
+import com.epam.random_coffee.events.api.request.UpdateEventRequest
+import com.epam.random_coffee.events.api.response.EventResponse
+import com.epam.random_coffee.events.model.Event
+
+import scala.concurrent.Future
+
+trait EventRepository {
+  def save(eventName: String): Future[Unit]
+
+  def findByName(eventName: String): Future[Option[EventResponse]]
+
+  def findById(id: Int): Future[Option[Event]]
+
+  def refresh(id: Int, newEventName: UpdateEventRequest): Future[Unit]
```

Comment:
I used "refresh" name because the "update" method from doobie.util appears in the implementation of this.
Maybe it's better to use "updateEvent" name?

Reply:
Method signatures always come before their implementation, which means it's better not to change method names just because there's a problem with their implementation. Otherwise it's impossible to change an implementation without changing anyone noticing it.
If you have some technical issue, we can help you to deal with it.

Reply:
Thanks for the clarification, I will fix it.